packages/interface/src/components/complementary-area/index.js

```diff
@@ -144,6 +148,7 @@ function ComplementaryArea( {
 								isActive && ( ! showIconLabels || isLarge )
 							}
 							aria-expanded={ isActive }
```

Comment:
The `aria-expanded` attribute (and possibly others) should always get a boolean value instead of `undefined`.

Reply:
`undefined` is listed as one of the allowed values for the `aria-expanded` property, as it defines an "[element that does not own or control an expandable grouping element](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-expanded#values)." This is also validated by the [jsx-a11y/aria-proptypes](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/93f78856655696a55309440593e0948c6fb96134/docs/rules/aria-proptypes.md) rule.